app/view_models/nsm/v1/defendant_details.rb

```diff
@@ -29,45 +29,51 @@ def main_defendant_maat
         main_defendant['maat']
       end
 
+      def main_defendant_value
+        if main_defendant_maat.present?
+          multiline_text("#{main_defendant_name}\n#{main_defendant_maat}")
+        else
+          main_defendant_name
+        end
+      end
+
       def additional_defendants
         defendants.reject { |defendant| defendant['main'] == true }
       end
 
       def main_defendant_rows
         [
           {
-            title:  I18n.t(".nsm.claim_details.#{key}.main_defendant_name"),
-            value: main_defendant_name
-          },
-          (if main_defendant_maat.present?
-             {
-               title: I18n.t(".nsm.claim_details.#{key}.main_defendant_maat"),
-              value: main_defendant_maat
-             }
-           end)
+            title:  I18n.t(".nsm.claim_details.#{key}.main_defendant"),
+            value: main_defendant_value
+          }
         ]
       end
 
       def additional_defendant_rows
         additional_defendants.map.with_index do |defendant, index|
           [
             {
-              title: I18n.t(".nsm.claim_details.#{key}.defendant_name", count: index + 1),
-              value: construct_name(defendant)
-            },
-            (if defendant['maat'].present?
-               {
-                 title: I18n.t(".nsm.claim_details.#{key}.defendant_maat", count: index + 1),
-                value: defendant['maat']
-               }
-             end)
+              title: I18n.t(".nsm.claim_details.#{key}.additional_defendant", count: index + 2),
+              value: construct_value(defendant)
+            }
           ]
         end
       end
 
       def rows
         { title:, data: }
       end
+
+      private
+
+      def construct_value(defendant)
+        if defendant['maat'].present?
+          multiline_text("#{construct_name(defendant)}\n#{defendant['maat']}")
```

Comment:
It feels a little bit heavy-handed to modify the string twice - first to add a newline and then to convert the newline into a `<br>`. You could do it in one step with `safe_join([construct_name(defendant), tag.br, defendant['maat']])`

Reply:
updated